pkg/config/output.go

```diff
@@ -84,7 +86,22 @@ func (p *PipelineConfig) updateEncodedOutputs(req egress.EncodedOutput) error {
 		return errors.ErrInvalidInput("multiple stream outputs")
 	}
 	if stream != nil {
-		conf, err := p.getStreamConfig(types.OutputTypeRTMP, stream.Urls)
+		u, err := url.Parse(stream.Urls[0])
+		if err != nil {
+			return errors.ErrInvalidInput("malformed url")
+		}
+
+		var outputType types.OutputType
+		switch u.Scheme {
+		case "srt":
+			outputType = types.OutputTypeSRT
+		case "rtmp":
```

Comment:
We also need to test for "rtmps"

Reply:
Would that default to same output type as rtmp or should there be a new output type for rtmps ?

Reply:
All current output URL schemes: rtmp://, rtmps:// and mux:// should use OutputTypeRTMP.